modules/visxBidAdapter.js

```diff
@@ -84,16 +100,22 @@ export const spec = {
         }
         slot.parents.push({parent: bidsMap[uid], key: sizeId, uid});
       });
+
+      videoMinDuration.push(deepAccess(bid, 'params.video.minduration', null));
+      videoMaxDuration.push(deepAccess(bid, 'params.video.maxduration', null));
+      videoSkip.push(deepAccess(bid, 'params.video.skip', null));
```

Comment:
Is there any reason to not read these values from `mediaTypes.video` ?

Reply:
Hi @Fawke,

We placed them not in `mediaTypes.video` because these parameters are not mentioned in the doc: https://docs.prebid.org/dev-docs/adunit-reference.html#adunitmediatypesvideo

Should we move them to `mediaTypes.video`?

Reply:
See https://github.com/prebid/Prebid.js/issues/5237#issue-618450834

Reply:
We weren't aware that page was out of date,  will fix today,  thank you!

Reply:
@Fawke, can you please provide details how to reproduce CORS error? Or copy request as curl command from chrome devtools? It shouldn't throw a CORS violation, thanks. As for Open RTB params I'll move them to the mediaTypes, thanks for pointing this out.

Reply:
Also there is test page on codepen: https://codepen.io/mk9/pen/XWpGGPa

Reply:
I had updated adapter, docs & codepen: video params has been moved from `bid.params` to `mediaTypes.video`.

Reply:
Hey, can you give us more info about the CORS error?
Our endpoint for header bidding requests returns both CORS-related headers. It can be checked on the Codepen page provided above:
```
Access-Control-Allow-Credentials: true
Access-Control-Allow-Origin: https://cdpn.io
```

We'd like to determine the problem. If changes are needed in our SSP, we want to schedule them as early as possible.

Reply:
Could be related to #6698